packages/manager/modules/product-offers/src/pricing/pricing.class.js

```diff
@@ -46,4 +46,8 @@ export default class Pricing {
   hasExtraPricing() {
     return !!this.extraPricing;
   }
+
+  mustBeDisplayed() {
```

Comment:
This shouldn't be part of pricing class. "Must be displayed" in which context ? This has no meaning at the product level.

The method should be called something like ```hasRenew ``` regarding it's body

Reply:
Sure, the method is renamed